frontend/src/App.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Routes from "./Routes";
+import { updateAccessToken } from "./utils/updateAccessToken";
+import Backdrop from "@material-ui/core/Backdrop";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    backdrop: {
+      zIndex: theme.zIndex.drawer + 1,
+      color: "#fff",
+    },
+  })
+);
+
+const App: React.FC = () => {
+  const classes = useStyles();
+  const [loading, setLoading] = React.useState(true);
+  React.useEffect(() => {
+    updateAccessToken(setLoading);
+  });
+  // if (loading) {
```

Comment:
should remove the comments or explain why they are present there.

Reply:
Yeah, I forgot to remove comments.